Directory.Packages.props

```diff
@@ -114,7 +114,7 @@
     <PackageVersion Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.6.0-beta.3" />
     <PackageVersion Include="OpenTelemetry.Instrumentation.Http" Version="1.7.0" />
     <PackageVersion Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.7.0" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.Process" Version="0.5.0-beta.4" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.Process" Version="1.0.0-alpha.6" />
```

Comment:
This is interesting. I don't see this version. https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Process/

![image](https://github.com/dotnet/aspire/assets/8291187/c36afd0a-2647-4556-94a4-b15157c9e42c)

Reply:
Same as above, the sources that are grepped are feeds in nuget.config.

Reply:
I think that is a problem. We shouldn't take new versions that aren't on nuget.org when we ship.